src/runtime.js

```diff
@@ -48,7 +48,7 @@ export async function serve(addr) {
     addr = { hostname, port: Number(port) };
   }
   const listener = Deno.listen(addr);
-  const host = `${listener.addr.hostname}:${listener.addr.port}`;
+  const host = `localhost:${listener.addr.port}`;
```

Comment:
The hostname can be set using `--addr 192.168.0.1:8080` so instead of replacing this string here with a hardcoded `localhost` it would be a good idea to set a default value for `hostname` on line 47